querybook/server/datasources/github.py

```diff
@@ -66,36 +66,20 @@ def link_datadoc_to_github(
         return github_link.to_dict()
 
 
-@register("/github/datadocs/<int:datadoc_id>/unlink/", methods=["DELETE"])
-def unlink_datadoc_from_github(datadoc_id: int) -> Dict:
-    with DBSession() as session:
-        datadoc = datadoc_logic.get_data_doc_by_id(datadoc_id, session=session)
-        api_assert(
-            datadoc is not None,
-            "DataDoc not found",
-            status_code=RESOURCE_NOT_FOUND_STATUS_CODE,
-        )
-        assert_can_write(datadoc_id, session=session)
-        verify_data_doc_permission(datadoc_id, session=session)
-
-        logic.delete_repo_link(datadoc_id)
-        return {"message": "Repository unlinked successfully"}
-
-
 @register("/github/datadocs/<int:datadoc_id>/is_linked/", methods=["GET"])
-def is_datadoc_linked(datadoc_id: int) -> Dict[str, bool]:
+def is_datadoc_linked(datadoc_id: int) -> Dict[str, Optional[str]]:
     with DBSession() as session:
```

Comment:
iirc we dont really need the with `DBSession() as session:` for APIs, the `with_session` will get the same session automatically under the same request context